spec/models/item_spec.rb

```diff
@@ -152,5 +152,18 @@
       expect(Item.find(item_4.id).active?).to eq(true)
       expect(Item.find(item_5.id).active?).to eq(true)
     end
+
+    it "activate_all_items" do
+      dog_shop = create(:merchant, enabled?: false)
+      item_1 = create(:item, active?: false, merchant: dog_shop)
+      item_2 = create(:item, active?: false, merchant: dog_shop)
+      item_3 = create(:item, active?: false, merchant: dog_shop)
+
+      dog_shop.items.activate_all_items
```

Comment:
I thought about it some more and I think these tests can safely use `Items.activate_all_items` with items from just one shop, as the point of this method is to change any items it's called on to activate (or in the other case, deactive) items. It's job is not to find the items it should be working on, that's our job to make that collection available.

Reply:
Oooh, yes. I like that reasoning. I'll go change it.